src/components/form/FSelect.vue

```diff
@@ -312,7 +312,11 @@ const { handleFocus, handleChange } = useInputEventBindings(
 );
 
 // Any change on the field value should trigger validation
-watch(fieldValue, handleChange);
+watch(fieldValue, value => {
+  // When resetting to an empty value, we don't want to trigger validation
+  if (value === '' || value === null || value === undefined) return;
+  handleChange(value);
+});
```

Comment:
Maybe "handleChange" could be replaced by `validate` from useFieldWithValidation. What do you think?
I find weird the way I've done this by the past (I mean, the coupling with useInputEventBindings)
```suggestion

watch(fieldValue, value => {
  // When resetting to an empty value, we don't want to trigger validation
  if (value === '' || value === null || value === undefined) return;
  validate(value);
});
```